tests/test_9_surfaces.py

```diff
@@ -12,57 +12,57 @@
 def test_contour_surface():
     # # circle contour
     # assert_almost_equal(ImmutMorphology(os.path.join(_path, "circle_contour.asc")).soma.surface,
-    #                     np.pi, places=2)
+    #                     np.pi, decimal=2)
     # assert_almost_equal(Morphology(os.path.join(_path, "circle_contour.asc")).soma.surface,
-    #                     np.pi, places=2)
+    #                     np.pi, decimal=2)
 
     # # triangular contour
     # assert_almost_equal(ImmutMorphology(os.path.join(_path, "h5/v1/Neuron.h5")).soma.surface,
-    #                     0.01, places=5)
+    #                     0.01, decimal=5)
     # assert_almost_equal(Morphology(os.path.join(_path, "h5/v1/Neuron.h5")).soma.surface,
-    #                     0.01, places=5)
+    #                     0.01, decimal=5)
 
     # # single point ASC
     # assert_almost_equal(ImmutMorphology(os.path.join(_path, "simple.asc")).soma.surface,
-    #                     4 * np.pi, places=2)
+    #                     4 * np.pi, decimal=2)
     # assert_almost_equal(Morphology(os.path.join(_path, "simple.asc")).soma.surface,
-    #                     4 * np.pi, places=2)
+    #                     4 * np.pi, decimal=2)
```

Comment:
do you need this ?

Reply:
I don't know. I just kept what was before.